modules/criteoBidAdapter.js

```diff
@@ -445,7 +446,7 @@ function buildCdbRequest(context, bidRequests, bidderRequest) {
       const slot = {
         impid: bidRequest.adUnitCode,
         transactionid: bidRequest.transactionId,
-        auctionId: bidRequest.auctionId,
+        auctionId: bidderRequest.ortb2?.source?.tid || generateUUID(),
```

Comment:
Please make sure you're not passing this to dsps if you pass openrtb. This wouldn't comply with openrtb

Reply:
It won't be but indeed ; I've amended my change to make it completely obvious & transparent and rely exclusively on a random uuid